esx_service/vmdk_ops.py

```diff
@@ -153,12 +157,24 @@ def RunCommand(cmd):
 
     return (s, o)
 
+def is_valid_volume_name(name):
+    """ Check the given name is a valid volume name """
+    pattern = "^" + VALID_VOLUME_NAME_REG + "$"
+    valid_name_reg = re.compile(pattern)
+    if valid_name_reg.match(name):
```

Comment:
Nit: Same as above - we can simplify it a little bit:

   return valid_name_reg.match(name)